<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>635</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>50</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SystemControlMecânica - Menu Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(55, 57, 63);
color: rgb(229, 229, 229);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bemVindo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mercado</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>104</width>
      <height>46</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>244</width>
      <height>291</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;O que deseja fazer ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="button_cadastro">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 85, 94);</string>
      </property>
      <property name="text">
       <string>Cadastrar Clientes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="material">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(88, 92, 102);</string>
      </property>
      <property name="text">
       <string>Adicionar Material</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_estoque_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 85, 94);</string>
      </property>
      <property name="text">
       <string>Editar Material</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_produtos">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 85, 94);</string>
      </property>
      <property name="text">
       <string>Listar Material</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_orca">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 85, 94);</string>
      </property>
      <property name="text">
       <string>Realizar Orçamento</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_orca_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>244</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Registrar Finanças </string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_orca_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>244</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Consultar Restrições de Veículos - Detran </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
